apps/portals/src/portal-components/challengeportal/ChallengeDetailPageWrapper.tsx

```diff
@@ -11,7 +15,23 @@ import { AppUtils, ChallengeDetailPage } from 'synapse-react-client'
  */
 const ChallengeDetailPageWrapper = () => {
   const projectId = AppUtils.useQuerySearchParam('id')
-  if (projectId) {
+  const { data, isLoading } = SynapseQueries.useGetEntityBundle(
+    projectId,
+    undefined,
+    {
+      includeAnnotations: true,
+    },
+    {
+      enabled: !!projectId,
+    },
+  )
+  if (isLoading) {
+    return <></>
```

Comment:
Consider rendering a skeleton for the loading case

Reply:
It's not ideal in either case.  Either we render a skeleton, which is non-sensical for closed/upcoming challenges.  Or we render nothing, which is not ideal for active challenges.